mmv1/products/discoveryengine/ChatEngine.yaml

```
@@ -0,0 +1,175 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ChatEngine'
+description: |
+  Vertex chat and Conversation Enging Chat type
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Vertex AI Conversation': 'https://cloud.google.com/dialogflow/cx/docs/concept/generative'
+  api: 'https://cloud.google.com/generative-ai-app-builder/docs/reference/rest/v1alpha/projects.locations.collections.engines'
```

```suggestion
  api: 'https://cloud.google.com/generative-ai-app-builder/docs/reference/rest/v1/projects.locations.collections.engines'
```

Doesn't seem to exist yet but I assume this is where they'll be?

---

@melinath Yes we checked with the product team and the updated v1 doc will be published at this endpoint